docs/cargo_features.md

```diff
@@ -11,9 +11,6 @@ The default feature set enables most of the expected features of a game engine,
 
 |feature name|description|
 |-|-|
-|android-game-activity|Android GameActivity support. Default, choose between this and `android-native-activity`.|
-|android_shared_stdcxx|Enable using a shared stdlib for cxx on Android|
-|animation|Enable animation support, and glTF animation loading|
```

Comment:
Decided to do the same sorting as in `Cargo.toml` and move bevy crates to the top; more than happy to revert this change if preferred.